tests/schemas/FnrSchema.test.ts

```diff
@@ -1,7 +1,12 @@
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, vi } from "vitest";
 import { ZodError } from "zod";
 import { FnrSchema } from "@/schemas/nærmestelederFormSchema";
 
+vi.mock("@/env-variables/envHelpers", () => ({
+  // We want to test the production validation logic too
+  isNonProd: false,
```

Comment:
Liten input angående navngiving og logikk for "isNonProd", vil kanskje anbefale å endre til isProd, pga
- slipper dobbelt negasjon "not not prod" (isNonProd == false)
- Slipper uheldige feil hvis det f.eks kommer en annen runtime_env (ala DEMO2) ["local", "demo", "dev", "test"].includes, slik at det da blir tolket til å være prod.